### DISCUSSION TITLE
Quasar Dev SSR with Vite is incompatible with https: true?

### DISCUSSION THREAD

**Discussion by rodrigoslayertech** [quasarframework/quasar#14353] [CLI - SSR mode]
I'm trying to activate `https` in `quasar dev-m ssr`, but I'm not getting it. It only opens in the http.

My conf:

![image](https://user-images.githubusercontent.com/9668277/189392402-8864398a-e43c-4c27-b577-f2d276211765.png)

Using `quasar dev` works fine, but in SSR mode, not.

I'm using Vite.

**Comment by george-dragnea** [comment]
Did you manage to make it work? On webpack it worked, but on vite, I can't figure it out.

**Reply by rodrigoslayertech** [reply]
I can't using Vite.

**Comment by george-dragnea** [comment]
Any updates on this?
